packages/s/sunshine/package.yml

```diff
@@ -33,17 +35,15 @@ builddeps  :
     - nodejs
 setup      : |
     %patch -p1 -i $pkgfiles/0001-package-linux-Provide-launchable-tags-in-appstream-m.patch
-    %patch -p1 -i $pkgfiles/ftbfs-upnpc.patch
-    pushd src_assets/common/assets/web
-        npm install
-    popd
+
     %cmake_ninja \
+                -DCMAKE_BUILD_TYPE=Release \
+                -DBUILD_DOCS=OFF \
+                -DBUILD_TESTS=OFF \
                 -DSUNSHINE_ASSETS_DIR=share/sunshine/assets \
                 -DSUNSHINE_EXECUTABLE_PATH=/usr/bin/sunshine
```

Comment:
For my local build I also had to add `-DCUDA_FAIL_ON_MISSING=OFF`, probably because I don't have a nvidia card. Will this be a problem on the build server?

Reply:
Seems like they started to fail the build if no CUDA is detected, so I added `-DSUNSHINE_ENABLE_CUDA=OFF`